### PR TITLE
7.0.x Synapse to Ballerina Initial Code changes and updating ballerina version to 0.92 

### PR DESCRIPTION
**Pull request by arukshani**

### CODE REVIEW

File: tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/ballerinahelper/HttpClientConnector.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+package org.wso2.ei.tools.converter.common.ballerinahelper;
```
Comment (a5anka) [inline tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/ballerinahelper/HttpClientConnector.java]: Shall we add the missing license header?

Comment (a5anka) [inline tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/ballerinahelper/HttpClientConnector.java]: Let's fix in the other places as well.

File: tools/converter/ds2ballerina/src/main/java/org/wso2/ei/tools/ds2ballerina/util/Util.java [hunk 2]
```diff
@@ -57,7 +57,7 @@ public static void createConfigProperties(BLangModelBuilder modelBuilder, DataSe
             for (Map.Entry<String, String> entry : configPropertiesMap.entrySet()) {
                 modelBuilder.createStringLiteral(null, null, entry.getKey());
                 modelBuilder.createStringLiteral(null, null, entry.getValue());
-                modelBuilder.addMapStructKeyValue(null, null);
+              //  modelBuilder.addMapStructKeyValue(null, null);
```
Comment (a5anka) [inline tools/converter/ds2ballerina/src/main/java/org/wso2/ei/tools/ds2ballerina/util/Util.java]: Shouldn't we remove the commented out code before merging?

File: tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/ballerinahelper/HttpClientConnector.java (hunk 1 above)
Comment (arukshani) [inline tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/ballerinahelper/HttpClientConnector.java]: This is fixed now.

File: tools/converter/ds2ballerina/src/main/java/org/wso2/ei/tools/ds2ballerina/util/Util.java (hunk 2 above)
Comment (arukshani) [inline tools/converter/ds2ballerina/src/main/java/org/wso2/ei/tools/ds2ballerina/util/Util.java]: I removed most of the unnecessary code comments but kept some of them as those need to be revisited later.

File: tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/ballerinahelper/Message.java [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+            ballerinaASTModelBuilder.createNameReference(Constant.BLANG_PKG_MESSAGES, Constant.BLANG_SET_JSON_PAYLOAD);
+
+        } else if (Constant.XML.equals((String) parameters.get(Constant.TYPE))) {
+            ballerinaASTModelBuilder.addTypes(Constant.BLANG_TYPE_XML); //type of the variable
+            ballerinaASTModelBuilder.addComment("//TODO: Change the double quotes to back tick. ");
+            ballerinaASTModelBuilder.createXMLLiteral((String) parameters.get(Constant.FORMAT));
+            ballerinaASTModelBuilder.createVariable((String) parameters.get(Constant.PAYLOAD_VAR_NAME), true); //name of
+            // the variable
+            ballerinaASTModelBuilder.createNameReference(Constant.BLANG_PKG_MESSAGES, Constant.BLANG_SET_XML_PAYLOAD);
+        }/* else if (Constant.STRING.equals((String) parameters.get(Constant.TYPE))) {
```
Comment (pamod) [inline tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/ballerinahelper/Message.java]: Shall we remove this commented line ?

File: tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/generator/CodeGenVisitor.java [hunk 4]
```diff
@@ -740,6 +744,11 @@ public void visit(AbortStmt abortStmt) {
     }
 
     @Override
+    public void visit(RetryStmt retryStmt) {
+
```
Comment (pamod) [inline tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/generator/CodeGenVisitor.java]: Should we throw not-implimented/unsupported exception ?

File: tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/ballerinahelper/Message.java (hunk 3 above)
Comment (arukshani) [reply tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/ballerinahelper/Message.java]: they were removed in a later commit. https://github.com/wso2/product-ei/commit/8eed353ae62790097571f73501170663b0bdb8f9#diff-609b8bd6feb33c1f80efc77990291592L85

File: tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/generator/CodeGenVisitor.java (hunk 4 above)
Comment (arukshani) [reply tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/generator/CodeGenVisitor.java]: Added NotImplemtedException. https://github.com/wso2/product-ei/pull/1216

**Pull request merged**